src/track/track.cpp

```diff
@@ -84,6 +84,9 @@ Track::Track(
                 << "->"
                 << numberOfInstancesBefore + 1;
     }
+
+    connect(this, &Track::titleChanged, this, &Track::infoChanged);
+    connect(this, &Track::artistChanged, this, &Track::infoChanged);
```

Comment:
I would prefer to call the additional signals explicit.
For less magic to consider in case of future refactoring.

Reply:
You want me to add slots that only contain `emit infoChanged()` ?

Reply:
No, I mean copy emit infoChanged() below emit artistChanged()

Than the call tree has one interruption less when debugging.

Reply:
Than we can consider to get rid of infoChanged() in favor of changed()

Reply:
No, the signal signature is incompatible.